### DISCUSSION TITLE
Tribe's Events Calendar

### DISCUSSION THREAD

**Discussion by drewvolz** [frog-pond/ccc-server#566] [Ideas]
Have we ever discussed using Tribe's Events Calendar json api? It has a ton more information. It could power additional views, give more context about events, and allow us to provide searching and filtering. We could even craft a link to an event as an `.ics` file. Lots of opportunities here!

* [HTML version](https://wp.stolaf.edu/calendar/events/)
* [Full calendar data (paged)](https://wp.stolaf.edu/calendar/wp-json/tribe/events/v1/events)
* [Detail event data](https://wp.stolaf.edu/calendar/wp-json/tribe/events/v1/events/48576)
* [Detail as iCal file](https://wp.stolaf.edu/calendar/event/paw-program-9/?ical=1) (take detail's url param and add `ical=1` param)

**Comment by hawkrives** [comment]
1. *!@# that plugin. 
2. That's a good idea. We should add TEC as a source for some calendar things. It would give us metadata about tags and categories, at least!
